dependencies.yaml

```diff
@@ -649,6 +649,7 @@ dependencies:
         packages:
           - dask-cuda==24.4.*
           - *numba
+          - dask-expr==0.4.0
```

Comment:
Latest version of dask-expr supported for dask 2024.1.1

Reply:
That is indeed my preference: https://github.com/rapidsai/rapids-dask-dependency/issues/23